scripts/cibuild.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+vagrant up --provision
+vagrant ssh -c "
+    cd /vagrant
+    echo -e '\nRunning linter...'
+
+    # run linter with console output
+    npm run lint
+
+    # run linter with file output for Jenkins
+    npm run lint:ci
```

Comment:
True. The `npm run lint:ci` could still be run in `scripts/cibuild` even if `npm run lint` is moved into `scripts/test`. That would probably be a better way to handle it rather than conditionals.